typhos/tests/conftest.py

```diff
@@ -230,7 +230,7 @@ def hints(self):
 
 @pytest.fixture(scope='function')
 def device():
-    dev = MockDevice('Tst:This', name='Simulated Device')
+    dev = MockDevice('Tst:This', name='simulated_device')
```

Comment:
It turns out that this name change is what breaks the suite tests. I'm a bit baffled, will look into it closer- is there a latent typhos bug that has been chilling?
Hopefully not... It turns out I didn't NEED to change this- it was part of my debug process as something that looked wrong.

Reply:
I found and fixed the related bug in https://github.com/pcdshub/typhos/pull/478/commits/6970d70f05edc481baac2931777c159f7996413b

It turns out that we store the "cleaned up" name only, so when I made the device have a "normal" name it was unable to find the panel.

Reply:
Great find!